src/test/php/web/rest/unittest/api/Monitoring.class.php

```diff
@@ -15,10 +16,7 @@ public function __construct() {
   }
  
   #[Get('/monitoring/systems')]
-  public function systems(
-    #[Request]
-    $req
-  ) {
+  public function systems(Request $req) {
     return Response::ok()->entity(['page' => $req->param('page')]);
```

Comment:
Example of how this make code easier in PHP7, where we have to put parameter annotations onto their own line!